pkg/ingester/client/client.go

```diff
@@ -3,12 +3,14 @@ package client
 import (
 	"flag"
 
+	"github.com/go-kit/kit/log"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 	"google.golang.org/grpc"
 	_ "google.golang.org/grpc/encoding/gzip" // get gzip compressor registered
 	"google.golang.org/grpc/health/grpc_health_v1"
 
+	_ "github.com/cortexproject/cortex/pkg/util/grpc/encoding/snappy" // get snappy compressor registered
```

Comment:
I think we should move import this import into `pkg/util/grpcclient`, where it is used. Importing it only here will not work for code that doesn't use `pkg/ingester/client`, but still uses `pkg/util/grpcclient`.